config/routes.rb

```diff
@@ -364,6 +364,7 @@
       get '/:step' => 'inherited_proofing#show', as: :step
       put '/:step' => 'inherited_proofing#update'
       get '/return_to_sp' => 'inherited_proofing#return_to_sp'
+      get '/errors/no_information' => 'inherited_proofing#no_information'
```

Comment:
With a nested route like this, typically I'd expect this to be broken out to a separate controller, rather than funneling all the routes through a single controller (e.g. `InheritedProofingErrorsController`).

Reply:
We probably will move this to the other ErrorController and Error handling work that Jeff is working on.

Reply:
Can you link me to the related ticket?